manual/contents/src/operations/configuration-reference.asciidoc

```diff
@@ -16,6 +16,9 @@ Each connector has a unique key to identify it, denoted `(bolt-connector-key)` i
 
 include::{importdir}/neo4j-config-docs-docs-jar/ops/configuration-bolt-connector-attributes.asciidoc[]
 
+[CAUTION]
+Using autogeneration of self-signed SSL certificates will not work if the Neo4j server has been configured with multiple connectors that do not bind to the same IP address.
+If you need to use multiple IP addresses, please configure certificates manually and consider using multi-host or wildcard certificates instead.
```

Comment:
This and the next change are in the right location. Pls just change the wording according to the above.